core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -43,8 +43,18 @@
      * Get the IP address that this container may be reached on (may not be the local machine).
      *
      * @return an IP address
+     * @see #getHost()
      */
     default String getContainerIpAddress() {
+        return getHost();
+    }
+
+    /**
+     * Get the host that this container may be reached on (may not be the local machine).
```

Comment:
```suggestion
     * Get the IP address of the host that this container may be reached on (may not be the local machine).
```

Reply:
it is not always API (e.g. sometimes it is "localhost" or even "docker.corp.org")